addons/web/static/src/js/views/calendar/calendar_controller.js

```diff
@@ -426,10 +426,10 @@ var CalendarController = AbstractController.extend({
     },
     /**
      * @private
-     * @param {OdooEvent} event
+     * @param {OdooEvent} ev
      */
-    _onPrev: function () {
-        event.stopPropagation();
+    _onPrev: function (ev) {
+        ev.stopPropagation();
```

Comment:
Could also have just added the `event` parameter, so the other two lines didn't have to change ;)

Reply:
This was made to avoid the confusion between an `OdooEvent` and a JavaScript `Event`